core/object/object.h

```diff
@@ -215,6 +215,7 @@ enum MethodFlags {
 	METHOD_FLAG_VARARG = 16,
 	METHOD_FLAG_STATIC = 32,
 	METHOD_FLAG_OBJECT_CORE = 64,
+	METHOD_FLAG_VIRTUAL_REQUIRED = 128,
```

Comment:
@reduz Can you confirm whether it's fine to add a new method flag?

Reply:
Tried to ask three times but no luck, so let's merge and reduz will tell us if this was incorrect :)